app/models/mixins/reportable_mixin.rb

```diff
@@ -150,23 +150,19 @@ def add_includes(data_records, options)
       end
 
       if association == "categories"
-        association_objects = []
-        assochash = {}
+        association_hash = {}
         includes["categories"][:only].each do|c|
           entries = Classification.all_cat_entries(c, self)
           entarr = []
           entries.each { |e| entarr.push(e.description) }
```

Comment:
probably this can also be renamed from `entarr` to `descriptions` and the code simplified to:
```ruby
descriptions = entries.collect(&:description)
```